src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoMockTest.java

```diff
@@ -43,63 +49,77 @@
 import com.amazonaws.services.dynamodbv2.model.ComparisonOperator;
 import com.amazonaws.services.dynamodbv2.model.Condition;
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 
-public class DynamoSchedulePlanDaoMockTest {
+public class DynamoSchedulePlanDaoMockTest extends Mockito {
```

Comment:
Still need the following tests

* createSchedulePlan sets the studyKey
* updateSchedulePlan sets the studyKey

Reply:
updateSchedulePlan nullifies the key of the passed in plan, and verifies it is set; this wasn't being done in the createSchedulePlan test, despite testing the setting of several other fields. I've added that now.